test/cpp/api/nn_utils.cpp

```diff
@@ -255,3 +187,431 @@ TEST_F(NNUtilsTest, ConvertParameters) {
     ASSERT_TRUE(torch::equal(sample.data(), vec.data().slice(0, 0, 5)));
   }
 }
+
+int64_t PackedSequenceTest_batch_size = 5;
+int64_t PackedSequenceTest_max_length = 6;
+
+std::vector<torch::Tensor> PackedSequenceTest_ordered_sequence(torch::ScalarType tensor_type) {
+  std::vector<torch::Tensor> seqs;
+  seqs.reserve(PackedSequenceTest_batch_size);
+  for (int64_t i = 0; i < PackedSequenceTest_batch_size; i++) {
+    seqs.emplace_back(torch::empty({
+      torch::randint(1, PackedSequenceTest_max_length, {1}).item<int64_t>()
+    }, tensor_type));
+  }
+  for (auto& s : seqs) {
+    s.random_(-128, 128);
+  }
+  sort(
+    seqs.begin(),
+    seqs.end(),
+    [&](const torch::Tensor& t1, const torch::Tensor& t2) {
+      return t1.size(0) > t2.size(0);
+    }
+  );
+  return seqs;
+}
+
+std::tuple<torch::Tensor, torch::Tensor> PackedSequenceTest_padded_sequence(torch::ScalarType tensor_type) {
+  // Create Tensor of random padded sequences
+  auto ordered = PackedSequenceTest_ordered_sequence(tensor_type);
+  auto lengths = torch::empty({ordered.size()}, torch::kInt64);
```

Comment:
..\test\cpp\api\nn_utils.cpp(218): error C2398: Element '1': conversion from 'unsigned __int64' to '_Ty' requires a narrowing conversion